packages/nx-plugin/src/generators/plugin/plugin.spec.ts

```diff
@@ -128,6 +128,48 @@ describe('NxPlugin Plugin Generator', () => {
     ).toContain('const variable = "<%= projectName %>";');
   });
 
+  it('should not create generator and executor files for minimal setups', async () => {
+    await pluginGenerator(tree, getSchema({ name: 'myPlugin', minimal: true }));
+
+    [
+      'libs/my-plugin/project.json',
+      'libs/my-plugin/generators.json',
+      'libs/my-plugin/executors.json',
```

Comment:
We talked about these 2 being non-existant, do we want to keep them or remove them? I know we left off w/o a firm decision.